components/build-image.md

```diff
@@ -0,0 +1,18 @@
+# Building images
```

Comment:
Maybe just call this README.md so that it's rendered in github when browsing this directory?